### DISCUSSION TITLE
Livewire wire:ignore is not working with this package.

### DISCUSSION THREAD

**Discussion by JeffGepiga** [abordage/laravel-html-min#20] [General]
Please help fix this bug. Ty

**Comment by jeffegiovani** [comment]
I also had problems with `wire:ignore`.
A good ideia was using persistent middleware, I tried but it didn't work (Laravel 11.13 with Livewire 3.5.1).

Using the `#[Lazy]` _(Livewire\Attributes\Lazy)_, was the solution I found.
